docs/src/main/sphinx/security/oauth2.rst

```diff
@@ -108,6 +108,12 @@ The following configuration properties are available:
        for more information.
    * - ``http-server.authentication.oauth2.principal-field``
      - The field of the access token used for the Trino user principal. Defaults to ``sub``. Other commonly used fields include ``sAMAccountName``, ``name``, ``upn``, and ``email``.
+   * - ``oauth2-jwk.http-client.http-proxy``
```

Comment:
Why only these two? Both of them come from airlift http client configuration which provides many more useful options (https://github.com/airlift/airlift/blob/master/http-client/src/main/java/io/airlift/http/client/HttpClientConfig.java) which probably should also be documented. Trino uses quite a few instances of http client, each with it's own prefix. How about documenting all of http client configuration properties in one place and just referencing it here?

Reply:
I was thinking about the same ? Should it be a separate documentation for HTTP client configuration ?